content/en/docs/tasks/traffic-management/locality-load-balancing/single-cluster-lb/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Single Multi-Zone Cluster Load Balancer Standalone Example
+description: This example demonstrates the traffic shifting between zones within a single cluster.
+weight: 40
+keywords: [locality,load balancing]
+test: no
+owner: istio/wg-networking-maintainers
+---
+
+## Before you begin
+
+This guide requires that you have a Kubernetes cluster with any of the
+[supported Kubernetes versions:](/docs/releases/supported-releases#support-status-of-istio-releases) {{< supported_kubernetes_versions >}}.
```

Comment:
The supported Kubernetes versions is not really necessary as that is "required" for installing Istio as it is. If there's a requirement for a specific version of Kubernetes or higher, I'd call it out with an explanation of why but it looks like any supported version is fine so this can be dropped.